package.json

```diff
@@ -13,9 +13,12 @@
     "build": "rollup -c",
     "build:watch": "rollup -cw",
     "lint": "eslint src",
-    "start": "cd example && npm start",
+    "start": "run-p build:watch start:example",
```

Comment:
I had a problem with this as I had to manually `cd example && npm ci` in order to run the start script. We could consider either using workspaces, or postinstall/prepare hook in the root, or lift the dev deps to the root.

Reply:
I decided to use workspaces via https://github.com/camunda/form-playground/pull/1/commits/0e41d37302985e46fa049f32de3c667ab5136b1d